features/path_rules.feature

```diff
@@ -0,0 +1,84 @@
+@sig-network @conformance @release-1.19
+Feature: Path rules
+
+  An Ingress may define routing rules based on the request path.
+
+  If the HTTP request path matches one of the paths in the
+  Ingress objects, the traffic is routed to its backend service.
+
+  Background:
+    Given a new random namespace
```

Comment:
Can we combine this into one step

`Given an Ingress resource`

as it is the common case and will just be boilerplate

Reply:
Yes, I can see boilerplate being reduced for such simple cases:

```
Given an Ingress resource in a new random namespace
```

As well as more granular boilerplate steps for complex setups where multiple resources need to exist in the same namespace:
```
Given a new random namespace
Given a self-signed TLS secret
Given an Ingress resource
```